agent/engine/dockerstate/dockerstate_test.go

```diff
@@ -258,3 +258,44 @@ func TestRemoveNonExistingImageState(t *testing.T) {
 		t.Error("Error removing incorrect image state")
 	}
 }
+
+// TestAddContainer tests first add container with docker name and
+// then add the container with dockerID
+func TestAddContainer(t *testing.T) {
```

Comment:
Can you rename the test to be more descriptive? You can use what you have written in the comments as well. Example:`TestAddContainerNameAndID`.